tests/Tests/500__jupyterhub/jupyterhub-user-access.robot

```diff
@@ -0,0 +1,89 @@
+*** Settings ***
+Force Tags       Smoke  Sanity
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Library          DebugLibrary
+Library          JupyterLibrary
+Library          OpenShiftCLI
+Library          OperatingSystem
+Suite Setup      Special User Testing Suite Setup
+Suite Teardown   End Web Test
+
+*** Variables ***
+
+*** Test Cases ***
+Test User Not In JH Access Groups
+    [Tags]  Smoke
+    ...     PLACEHOLDER  #Category tags
+    ...     ODS-503
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Wait for RHODS Dashboard to Load
+    Launch JupyterHub From RHODS Dashboard Dropdown
+    Login Verify Not Allowed  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+Test User In JH Admin Group
+    [Tags]  Smoke
+    ...     PLACEHOLDER  #Category tags
+    ...     ODS-503
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Wait for RHODS Dashboard to Load
+    Launch JupyterHub From RHODS Dashboard Dropdown
+    Login Verify Admin Allowed  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+Test User In JH Users Group
+    [Tags]  Smoke
+    ...     PLACEHOLDER  #Category tags
+    ...     ODS-503
+    Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+    Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+    Wait for RHODS Dashboard to Load
+    Launch JupyterHub From RHODS Dashboard Dropdown
+    Login Verify Regular User Allowed  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+
+*** Keywords ***
```

Comment:
I would put the keywords in one of the Resource files so that they can be reused by other tests